examples/md-flexible/src/Simulation.cpp

```diff
@@ -0,0 +1,573 @@
+/**
+ * @file Simulation.cpp
```

Comment:
The content here was not only moved but also changed! It might be easier to look at the commits touching this file and Simulation.h.